sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/README.md

```diff
@@ -0,0 +1,97 @@
+# Azure Event Hubs Checkpoint Store for Blob Storage
+
+<!-- cspell: ignore myapp -->
```

Comment:
Could also put it in your `.dict.txt` file so it doesn't break the flow of people reading the raw markdown e.g., in IDEs.